gustaf/spline/ffd.py

```diff
@@ -233,29 +224,32 @@ def _deform(self):
                     "spline or(and) the mesh are not yet defined. "
                     "Please set either spline or mesh."
             )
-        if self._is_calculated and self._is_calculated_trackable:
-            return None
+        if self._spline._data.get("gustaf_ffd_computed", False):
+           return 
+
+        spline = self._spline.copy()
```

Comment:
I think we can avoid this if we scale `_q_vertices` to be in spline's parametric space. was this one of our discussion points back then? we can talk about this for next PR